app/konnect/updates.md

```diff
@@ -3,11 +3,117 @@ title: Konnect Cloud Updates
+  Labels are key:value pairs, and are helpful for organizing, searching, and filtering subsets of {{site.konnect_short_name}} entities.
+
+**Teams and roles**
+: You can now manage {{site.konnect_short_name}} authorization with [teams and roles](/konnect/org-management/teams-and-roles).
+Existing RBAC roles have been converted to [predefined teams](/konnect/org-management/teams-and-roles/teams-reference), which are available for all {{site.konnect_short_name}} organizations.
+
+: Invite users and add them to these teams to manage user access. You can also assign individual roles per user.
+
+: With a {{site.konnect_short_name}}t Enterprise subscription, you can also [create custom teams](/konnect/org-management/teams-and-roles/manage) and assign per-entity permissions for each team.
+
+**Declarative configuration management support with decK**
+: As of [decK 1.12](https://github.com/Kong/deck/releases), standard decK commands such as `diff`, `sync`, and `dump` support {{site.konnect_short_name}} runtime groups.
+: Learn how to use decK with {{site.konnect_short_name}}:
+  * [Get started with decK and {{site.konnect_short_name}}](/deck/latest/guides/konnect)
+  * [Import](/konnect/getting-started/import) {{site.base_gateway}} or `konnect.konghq.com` configuration into `cloud.konqhq.com`
+  * [Manage runtime groups with decK](/konnect/runtime-manager/runtime-groups/declarative-config)
+
+**Tags for {{site.konnect_short_name}} services**
+: You can now connect {{site.konnect_short_name}} services to Gateway services with the [`_KonnectService` tag](/deck/latest/guides/konnect/#konnect-service-tags).
+
+**{{site.konnect_short_name}} UI/UX redesign**
+: The {{site.konnect_short_name}} UI now has reworked navigation, refreshed colors and styles, and a focus on consistent interactions throughout the application.
+
+: Navigation redesign details:
+  * Services is renamed to Service Hub
+  * Runtimes is renamed to Runtime Manager
+  * Shared Config is now part of Runtime Manager
+  * Dev Portal settings have moved from the general settings page to the Dev Portal section
+  * Refreshed iconography for the main menu
+
+**Dev Portal default theme**
+: The Dev Portal now comes with a new default theme: "Mint Rocket".
+
+**Dev Portal SSO support with your own IdP**
+: You can now configure custom identity providers (IdPs) for the Dev Portal.
+: See the [SSO documentation](/konnect/dev-portal/customization/#single-sign-on) for more information.
+
+**Simplified custom Dev Portal URL setup**
+: SSL certificate generation for custom URLs is now handled by {{site.konnect_short_name}}.
+
+: To set up a custom URL, enter your desired custom portal URL in the Dev Portal settings (for example, `portal.example.com`) and set up a CNAME record to the {{site.konnect_short_name}} organization's Dev Portal domain. See the [Dev Portal customization doc](/konnect/dev-portal/customization) for more information.
```

Comment:
```suggestion
: To set up a custom URL see the [Dev Portal customization documentation](/konnect/dev-portal/customization) for more information.
```
Simplifying here. The way it is currently written makes me feel like I can just enter any custom URL into Konnect and Konnect will do all of the registration for me within the application.